src/batch/batch.ts

```diff
@@ -0,0 +1,25 @@
+import { MessageType } from "../types/DSNP";
+import { NotImplementedError } from "../utilities/errors";
+
+export type BatchFileObject = string;
+
+/**
+ * createFile() takes a series of activity pub objects and returns a Batch file
+ * object for publishing. This method is not yet implemented.
+ *
+ * @params  events  An array of DSNPEvent to include in the batch file
+ * @returns         A batch file object
+ */
+export const createFile = (_events: MessageType[]): BatchFileObject => {
+  throw NotImplementedError;
+};
+
+/**
+ * openFile() takes a Batch file object . This method is not yet implemented.
+ *
+ * @params  file  An array of DSNPEvent to include in the batch file
```

Comment:
is this description correct? It seems like a copy and past error from above

Reply:
fixed